authentication/views.py

```diff
@@ -106,7 +108,8 @@ def get(self, request):
         try:
             payload = jwt.decode(token, settings.SECRET_KEY)
         except jwt.exceptions.DecodeError:
-            return HttpResponseRedirect(domain+'?verified_status=invalid_link')  # noqa
+            return HttpResponseRedirect(
+                domain + '?verified_status=invalid_link')  # noqa
```

Comment:
String formating is more pythonic here bro, 

something like  `'{}format this'.format(value)`

Reply:
That's not my feature. That was already done along time ago.